### DISCUSSION TITLE
How to create shared code between extensions?

### DISCUSSION THREAD

**Discussion by nodeworks** [directus/directus#14802] [Archive]
Is it possible to create a piece of code that can be reused by multiple extensions? Like helper functions etc, so they don't have to be copied over from extension to extension.

**Comment by br41nslug** [comment]
To be honest for small shared functions i tend to copy around that file as it's the least setup required. If the project reaches enough complexity then creating a local shared package is the way to go (like what directus does with `@directus/shared`) which you can link using `npm link` https://docs.npmjs.com/cli/v8/commands/npm-link
